### PR TITLE
Update topic tile text.

### PR DESCRIPTION
**Pull request by keyurva**

### CODE REVIEW

File: server/config/home_page/topics.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   {
     "id": "health",
     "title": "Health",
-    "description": "Dive into the realm of human well-being and explore the impact of health on societies.",
+    "description": "Explore Health Conditions, Health Equity, Helath Insurance, Mortality, and more.",
```
Comment (pradh) [inline server/config/home_page/topics.json:23]: Health

File: server/config/home_page/topics.json [hunk 2]
```diff
@@ -47,10 +47,10 @@
   {
     "id": "equity",
     "title": "Equity",
-    "description": "Wield the power of data to create a more just and equitable world.",
+    "description": "Explore Economic Equity, Education Equity, Health Equity, Justice Equity and more.",
```
Comment (pradh) [inline server/config/home_page/topics.json:50]: Justice System Equity (Justice Equity is odd)

**Review by pradh — APPROVED** [review]

File: server/config/home_page/topics.json (hunk 1 above)
Comment (keyurva) [reply server/config/home_page/topics.json]: Done.

File: server/config/home_page/topics.json (hunk 2 above)
Comment (keyurva) [reply server/config/home_page/topics.json]: Done.

**Pull request merged**